dts/bindings/charger/ti,bq25180.yaml

```diff
@@ -28,3 +28,6 @@ properties:
       The value specified will be rounded down to the closest implemented
       value. If set to 0 (default) skip setting the charge current value at
       driver initialization.
+
+  constant-charge-voltage-max-microvolt:
+    default: 4200000
```

Comment:
It might be safer to make this a required rather than provide a default. 4.2V is safe for most lithium batteries, but not all. Given the nature of lithium batteries, safer is usually better :)

Reply:
Can do, I just used what was already the default value for the device. (And the reason for this whole pull request, because my battery does need a lower voltage.)